### DISCUSSION TITLE
up有空能否探讨一下.net单元测试相关的内容

### DISCUSSION THREAD

**Discussion by Jazzl198442** [BYJRK/DotNet-Discussions#78] [C# 技术交流]
例如几种比较主流的测试框架, 什么情况下使用单元测试, 好处是什么, 在每次有新的提交或者分支合并时, 如果有效利用已经创建的单元测试快速验证新的代码逻辑是否存在问题等, 如何利用单元测试尽量完整的穷举代码分支以保证测试的全面性

**Comment by oufu99** [comment]
支持,感觉这个功能也挺需要的
